packages/webapp/src/components/RouterTab/Tab.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import { ReactNode, CSSProperties } from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.scss';
+
+export enum VARIANTS {
```

Comment:
NIT, but I think usually the convention for TS enums is to name them in singular and with PascalCase, so in this case `enum Variant`

Reply:
Oh, thank you so much!! I'll update it!